src/modules/search.js

```diff
@@ -172,6 +178,7 @@ class Search {
    * @param {string[]} [parameters.hiddenFacets] - Hidden facet fields to return
    * @param {object} [parameters.variationsMap] - The variations map object to aggregate variations. Please refer to https://docs.constructor.io/rest_api/variations_mapping for details
    * @param {object} [parameters.preFilterExpression] - Faceting expression to scope search results. Please refer to https://docs.constructor.io/rest_api/collections/#add-items-dynamically for details
+   * @param {object} [parameters.qsParam] - Parameters listed above can be serialized into a JSON object and parsed through this parameter. Please refer to https://docs.constructor.io/rest_api/search/queries for details
```

Comment:
GH doesn't let me comment on `getVoiceSearchResults` but can we add this to that function as well?